python/pip_install/tools/wheel_installer/wheel_installer.py

```diff
@@ -289,6 +298,17 @@ def _generate_build_file_contents(
     )
 
 
+def _generate_bin_bzl_contents(entrypoints: dict[str, str]) -> str:
+    return textwrap.dedent(
+        """\
+        # Autogenerated by wheel_installer.py
+        bin = struct(**{})
+        """.format(
+            repr({name: Label(target=target) for name, target in entrypoints.items()})
```

Comment:
Please use a dedicate formatting function instead of relying on repr(). The output of Python's repr isn't guaranteed to be stable between Python releases, nor is it guaranteed to be compatible with Starlark.